dotCMS/src/main/java/com/dotmarketing/servlets/ShortyServlet.java

```diff
@@ -321,8 +321,12 @@ private void doForward(final HttpServletRequest request,
 
       final StringBuilder pathBuilder = new StringBuilder(path)
               .append(id).append("/byInode/true");
+        //This logic is to get the filters to apply in the order were sent
+        final boolean isFieldNameAtUri = id.split(StringPool.FORWARD_SLASH)[2].equalsIgnoreCase(fieldName);
```

Comment:
is not dangerous to think it is always at least 2 elements?

Reply:
yes, it always comes the id and the fieldName